libs/features/jarvis-chat.js

```diff
@@ -215,6 +215,7 @@ const startInitialization = async (config, event, onDemand) => {
     [language, region] = config.locale.ietf.split('-');
   } else {
     [region, language] = config.locale.prefix.replace('/', '').split('_');
+    if (region === 'africa') region = 'ZA';
```

Comment:
Are there any other regions where this is the case? Then we have an array that we check and place it at the top of the file.
We should have the historic knowledge from AEM 🤔

Reply:
Currently I don't know any other region but we will check this. Thanks @mokimo
here is the [aem mapping](https://git.corp.adobe.com/wcms/hawks/blob/develop/app/jcr_root/conf/global/_sling_configs/jarvisLocaleConfig/.content.xml) if i'm right